llvm/test/CodeGen/RISCV/GlobalISel/instruction-select/select32.mir

```diff
@@ -0,0 +1,55 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
```

Comment:
This does not match the naming of the rest of the test files in this directory. Should be select-rv32.mir

Reply:
The tests for constant and copy don't follow this naming structure. We might want to change that in the future.